changelog.py

```diff
@@ -122,7 +122,7 @@ def collect_changelog(repo: Repository, milestone: Milestone):
     issues = repo.get_issues(milestone=milestone, state="all")
 
     comment_pattern = re.compile("<!--.*-->", re.RegexFlag.DOTALL)
-    changelog_description_pattern = re.compile("changelog:(?P<description>([^\n]+\n?)*)")
+    changelog_description_pattern = re.compile("Changelog:(?P<description>([^\n]+\n?)*)")
```

Comment:
Unfortunately, it's not enough because:
- we need to parse changelog text created in the previous style (as mentioned in https://github.com/intellij-rust/intellij-rust.github.io/pull/131#issuecomment-900159115) to support already created PRs
- in https://github.com/intellij-rust/intellij-rust/pull/7688, you use header for changelog section. Usually, people add new line (or even two new lines) symbols after header to write paragraph text on new line. The current pattern doesn't support such cases